index.html

```diff
@@ -29,9 +29,120 @@ <h1>Felipe<br>Oyarzo</h1>
                 </p>
             </div>
             <button class="button" type="button" id="btnHeadline">Start collaboration</button>
-            <div class="geometric"></div>            
+            <div class="geometric"></div>
```

Comment:
Kindly. Layout images should be placed in the background of your site, I recommend using background-image.

Reply:
hello @mgmediaweb , thanks for the feedback. I don't understand, the geometric shapes are already set to background-image for the div with the class geometric... which image should be in the background? Thank you kindly for your response

Reply:
@mgmediaweb I would really appreciate it if you could clarify your review please since I have to submit the code by today but still don't know what I should change. Thank you in advance

Reply:
I mean that the images must be in the bottom of the BODY or of the SECTION, not in floating DIV